src/rsz/src/Resizer.i

```diff
@@ -58,15 +57,7 @@ ensureLinked();
 
 namespace sta {
 
-// Defined in StaTcl.i
-LibertyCellSeq *
-tclListSeqLibertyCell(Tcl_Obj *const source,
-                      Tcl_Interp *interp);
-PinSet *
-tclListSetPin(Tcl_Obj *source,
-              Tcl_Interp *interp);
-
-using TmpNetSeq = NetSeq ;
+using TmpNetSeq = NetSeq;
 using TmpPinSet = PinSet;
```

Comment:
Why are these required?  StaTclTypes.i already handles NetSeq and PinSet.

Reply:
Different conversion rules attached to those: `TmpNetSeq*` gets freed by the swig conversion rule, `NetSeq*` does not. Let me add a comment